src/query/config/src/config.rs

```diff
@@ -1656,6 +1656,9 @@ pub struct QueryConfig {
     #[clap(long, value_name = "VALUE", default_value = "0")]
     pub cloud_control_grpc_timeout: u64,
 
+    #[clap(long)]
+    pub enable_queries_executor: bool,
```

Comment:
This config is not used.

Reply:
used here to switch executor: 

https://github.com/datafuselabs/databend/blob/8693b2f95b5beb3baa910d1e556040c43a384fc4/src/query/service/src/global_services.rs#L135-L139

I hard-encoded to test convinently, will use it before PR merged.

Reply:
Got it.
BTW, the PR in `draft` state still can do all the benchmark, and it's more safe.
If a PR is `review in ready`, it means all codes ready and can be reviewed :)

Reply:
Before this PR merged, I will disable the `QueriesPipelineExecutor`. It should not affect currently using executor.

Reply:
> Got it. BTW, the PR in `draft` state still can do all the benchmark, and it's more safe. If a PR is `review in ready`, it means all codes ready and can be reviewed :)

Thanks for your warm reminding :), will do next time.